samplesheets/vueapp/tests/unit/IrodsStatsBadge.spec.js

```diff
@@ -2,13 +2,17 @@ import { createLocalVue, mount } from '@vue/test-utils'
 import { projectUuid } from '../testUtils.js'
 import IrodsStatsBadge from '@/components/IrodsStatsBadge.vue'
 import '@/filters/prettyBytes.js'
+import fetch from 'node-fetch'
```

Comment:
This should not be used, `node-fetch` breaks stuff (see e.g. the currently breaking CI tests) and it is not needed. Instead, just replace the `updateStats()` method with a stub. See `SheetTableHeader.spec.js` for how the setup is done there.